### PR TITLE
cmake: Remove "find_package_handle_standard_args" warnings.

### PR DESCRIPTION
**Pull request by drmpeg**

### CODE REVIEW

**Comment by drmpeg** [comment]
Fixes https://github.com/gnuradio/gnuradio/issues/3581

Except for VOLK (which can easily be fixed) and UHD (which I'm not sure about).

**Review by mormj — APPROVED** [review]

File: cmake/Modules/FindCodec2.cmake [hunk 1]
```diff
@@ -9,7 +9,9 @@
 # Check for system libcodec2 #
 ##############################
 
-INCLUDE(FindPkgConfig)
+if(NOT PKG_CONFIG_FOUND)
+  INCLUDE(FindPkgConfig)
```
Comment (noc0lour) [inline cmake/Modules/FindCodec2.cmake:13]: is it possible to `unscreamify` this as well?

File: cmake/Modules/FindFFTW3f.cmake [hunk 2]
```diff
@@ -3,7 +3,9 @@
 
 # Find single-precision (float) version of FFTW3
 
-INCLUDE(FindPkgConfig)
+if(NOT PKG_CONFIG_FOUND)
+    INCLUDE(FindPkgConfig)
```
Comment (noc0lour) [inline cmake/Modules/FindFFTW3f.cmake:7]: lowercase

File: cmake/Modules/FindCodec2.cmake [hunk 3]
```diff
@@ -63,7 +65,7 @@ if(LIBCODEC2_INCLUDE_DIR AND LIBCODEC2_LIBRARIES)
   endif()
 endif(LIBCODEC2_INCLUDE_DIR AND LIBCODEC2_LIBRARIES)
 
-FIND_PACKAGE_HANDLE_STANDARD_ARGS(LIBCODEC2 DEFAULT_MSG LIBCODEC2_LIBRARIES LIBCODEC2_INCLUDE_DIRS)
+FIND_PACKAGE_HANDLE_STANDARD_ARGS(Codec2 DEFAULT_MSG LIBCODEC2_LIBRARIES LIBCODEC2_INCLUDE_DIRS)
```
Comment (noc0lour) [inline cmake/Modules/FindCodec2.cmake:68]: is lowercase possible?

File: cmake/Modules/FindGSL.cmake [hunk 4]
```diff
@@ -16,7 +16,9 @@
 #  GSL_LINK_DIRECTORIES = link directories, useful for rpath on Unix
 #  GSL_EXE_LINKER_FLAGS = rpath on Unix
 
-INCLUDE(FindPkgConfig)
+if(NOT PKG_CONFIG_FOUND)
+    INCLUDE(FindPkgConfig)
```
Comment (noc0lour) [inline cmake/Modules/FindGSL.cmake:20]: lowercase

File: cmake/Modules/FindZeroMQ.cmake [hunk 5]
```diff
@@ -1,4 +1,6 @@
-INCLUDE(FindPkgConfig)
+if(NOT PKG_CONFIG_FOUND)
+    INCLUDE(FindPkgConfig)
```
Comment (noc0lour) [inline cmake/Modules/FindZeroMQ.cmake:2]: `unscreamify` whole file?

File: cmake/Modules/FindTHRIFT.cmake [hunk 6]
```diff
@@ -1,4 +1,6 @@
-INCLUDE(FindPkgConfig)
+if(NOT PKG_CONFIG_FOUND)
+  INCLUDE(FindPkgConfig)
```
Comment (noc0lour) [inline cmake/Modules/FindTHRIFT.cmake:2]: lowercase

File: cmake/Modules/FindSNDFILE.cmake [hunk 7]
```diff
@@ -1,4 +1,6 @@
-INCLUDE(FindPkgConfig)
+if(NOT PKG_CONFIG_FOUND)
+    INCLUDE(FindPkgConfig)
```
Comment (noc0lour) [inline cmake/Modules/FindSNDFILE.cmake:2]: lowercase

File: cmake/Modules/FindGSM.cmake [hunk 8]
```diff
@@ -36,7 +38,7 @@ if(LIBGSM_INCLUDE_DIR AND LIBGSM_LIBRARIES)
   set(LIBGSM_LIBRARIES ${LIBGSM_LIBRARIES} ${LIBGSM_LIBRARY})
 endif(LIBGSM_INCLUDE_DIR AND LIBGSM_LIBRARIES)
 
-FIND_PACKAGE_HANDLE_STANDARD_ARGS(LIBGSM DEFAULT_MSG LIBGSM_LIBRARIES LIBGSM_INCLUDE_DIRS)
+FIND_PACKAGE_HANDLE_STANDARD_ARGS(GSM DEFAULT_MSG LIBGSM_LIBRARIES LIBGSM_INCLUDE_DIRS)
```
Comment (noc0lour) [inline cmake/Modules/FindGSM.cmake:41]: lowercase?

File: cmake/Modules/FindGSL.cmake (hunk 4 above)
Comment (noc0lour) [inline cmake/Modules/FindGSL.cmake:20]: lowercase

File: cmake/Modules/FindGSM.cmake [hunk 9]
```diff
@@ -9,7 +9,9 @@
 # Check for system libgsm #
 ###########################
 
-INCLUDE(FindPkgConfig)
+if(NOT PKG_CONFIG_FOUND)
+  INCLUDE(FindPkgConfig)
```
Comment (noc0lour) [inline cmake/Modules/FindGSM.cmake:13]: lowercase

File: cmake/Modules/FindFFTW3f.cmake (hunk 2 above)
Comment (noc0lour) [inline cmake/Modules/FindFFTW3f.cmake:7]: lowercase

File: cmake/Modules/FindCodec2.cmake (hunk 3 above)
Comment (noc0lour) [inline cmake/Modules/FindCodec2.cmake:68]: lowercase?

File: cmake/Modules/FindCodec2.cmake (hunk 1 above)
Comment (noc0lour) [inline cmake/Modules/FindCodec2.cmake:13]: lowercase

**Review by noc0lour — APPROVED** [review]
I'm happy as is, if you feel the urge you can follow my comments to further reduce the amount of ALL UPPERCASE CMake commands since you touched the files anyway. But don't feel obliged.

**Comment by drmpeg** [comment]
The whole issue of case in CMake files is a big can of worms. As we can see with this behavior in CMake >= 3.17, even the case of the filename itself makes a difference.

There was some discussion started in the Volk project. See:

https://github.com/gnuradio/volk/issues/387

At this time, I'm going to leave this PR as is. But if we can all agree on coherent case standard, I'll be happy to update the entire tree in another PR.

@skoslowski 
@jdemel

**Pull request merged**